Makefile

```
@@ -110,6 +110,21 @@ run: go.build
 	@# To see other arguments that can be provided, run the command with --help instead
 	$(GO_OUT_DIR)/$(PROJECT_NAME) --debug
 
+dev: $(KIND) $(KUBECTL)
+	@$(INFO) Creating kind cluster
+	@$(KIND) create cluster --name=provider-gcp-dev
+	@$(KUBECTL) cluster-info --context kind-provider-gcp-dev
+	@$(INFO) Installing Crossplane CRDs
+	@$(KUBECTL) apply -k https://github.com/hasheddan/crossplane//cluster?ref=kustomlocal
```

```suggestion
	@$(KUBECTL) apply -k https://github.com/crossplane/crossplane//cluster?ref=master
```

Change required after merge of https://github.com/crossplane/crossplane/pull/1594